src/ClaimService.Business/Features/Claims/Commands/Create/CreateClaimCommand.cs

```diff
@@ -0,0 +1,35 @@
+﻿using LT.DigitalOffice.ClaimService.Business.Shared.Enums;
+using MediatR;
+using System;
+using System.ComponentModel;
+using System.ComponentModel.DataAnnotations;
+
+namespace LT.DigitalOffice.ClaimService.Business.Features.Claims.Commands.Create;
+
+public record CreateClaimCommand : IRequest<Guid>
+{
+  /// <summary>
+  /// Name of the claim.
+  /// </summary>
+  [Required]
+  [MaxLength(100, ErrorMessage = "Name must be shorter than 100 symbols.")]
```

Comment:
In the Terms of reference there is another values (name has limit 50 symbols, content has limit 500), also check edit validator